.github/workflows/rust.yml

```diff
@@ -0,0 +1,22 @@
+name: Rust
```

Comment:
Do we have other language to test here? If it runs only for PRs, can we name it `CI` maybe?

Reply:
fair. This and the two comments you made below are all default from Github's rust action. I made no modifications.

Reply:
I guess it's fine then? :smile:

Reply:
i split the difference and called it "Rust CI" :joy_cat: